src/package.json

```diff
@@ -36,6 +36,7 @@
     "cookie-parser": "^1.4.6",
     "cross-spawn": "^7.0.3",
     "ejs": "^3.1.9",
+    "ep_etherpad-lite": "file:",
```

Comment:
Why add this here? All the files in ./src are actually "ep_etherpad-lite" (and ./src/package.json is the package.json that describes this module), so doesn't this introduce some kind of self-referenced loop?

Reply:
True. I don't know how this was added. Maybe by npm link?

Reply:
I'll continue with testing on Windows today.

Reply:
Introduced by e440d3d945294830a2d2bb4ccdd2b27829380229